ui_testing/testcases/basic_tests/test02_articles.py

```diff
@@ -464,3 +464,124 @@ def test019_download_article_sheet(self):
             fixed_sheet_row_data = self.fix_data_format(values)
             for item in fixed_row_data:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    def test021_optional_fields(self):
+        """
+        New: Articles: Optional fields: User can hide/show any optional field in Edit/Create form
+
+        LIMS:4123
+        :return:
+        """
+        
+        ################
+        ## ARCHIVING ###
+        ################ 
+
+
+        # open the configuration
+        self.article_page.info('+ Open article configuration')
+        self.article_page.open_configuration()
+
+        # Archive the optional fields
+        self.article_page.toggle_archive_field(field_name='unit')
+        self.article_page.toggle_archive_field(field_name='comment')
+        self.article_page.toggle_archive_field(field_name='related_article')
```

Comment:
This gonna change the UI, then u need to restore the default status in the teardown method

Reply:
@0xIslamTaha As i understand, the tearDown method runs after each test case, and restoring the UI won't be necessary after each case, so my question is how to specify when to run the restoring function after specific test cases using tearDown ?